docs/book/v5/tutorials/create-book-module.md

```diff
@@ -522,7 +522,7 @@ declare(strict_types=1);
 
 namespace Api\Book\Handler;
 
-use Api\App\Handler\HandlerTrait;
+use Api\App\Handler\Hanuse Api\App\Handler\AbstractHandler;
```

Comment:
```suggestion
use Api\App\Handler\AbstractHandler;
```
Small typo to fix first.

Reply:
Sorry @alexmerlin i was using the webui editor :p

Reply:
Done